app/controllers/users_controller.rb

```diff
@@ -1,15 +1,17 @@
 class UsersController < ApplicationController
+  before_action :find_user, except: %i(index new create)
+  before_action :logged_in_user, only: %i(index edit update destroy)
+  before_action :correct_user, only: %i(edit update)
+  before_action :admin_user, only: %i(destroy)
```

Comment:
```suggestion
  before_action :admin_user, only: :destroy
```

cho 1 dòng trống dưới dòng 5 em

Reply:
before_action có tác dụng gì em

Reply:
before_action: là callback, theo em hiểu là một action hoặc một danh sách các action sẽ được thực thi trước